data.js

```diff
@@ -493,6 +493,24 @@ module.exports = [
       'Women In Tech'
     ],
   },
+  {
+    name: 'Jusn Fernandes',
```

Comment:
I guess that your name is Juan instead of Jusn, isn't it? Maybe you want to update it before the merge 🥰

Reply:
:) I blame the new split keyboard...